.github/workflows/typespec-requirement.yaml

```diff
@@ -28,9 +39,49 @@ jobs:
       - run: npm ls -a
         continue-on-error: true
 
+      # May set output 'spec-lifecycle-$(matrix.spec-type)' to 'brownfield'
       - run: |
           eng/scripts/TypeSpec-Requirement.ps1 `
             -BaseCommitish HEAD^ `
             -TargetCommitish HEAD `
             -SpecType ${{ matrix.spec-type }}
+        id: tsr-ps1
         shell: pwsh
+
+  edit-label:
+    name: Add/Remove Label "brownfield"
+
+    needs: typespec-requirement
+
+    if: ${{ success() }}
+
+    permissions:
+      pull-requests: write
+    
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          sparse-checkout: |
+            .github/workflows
```

Comment:
Is this why you put the script under workflows to keep this path minimized? Should we clone the eng folder and I suspect we might end up needing the root project.json at well at some point if we need to restore packages.

Reply:
It wasn't about this path specifically.  I just though it made the more sense to organize the `github-script` scripts under `.github/workflows` rather than `eng`.  I agree some checks might need the root `package.json` and to run `npm ci`, but these steps take enough time, I think they should be avoided if the check doesn't actually need them.